bloom-housing/bloom
# 1774 preferred unit (same as one from Sean's fork)

## Issue

Addresses #1774


## Description

Updates both frontend and backend to save `unitType` id's (they come from the listing `units` property) instead of strings like `studio` in the `preferredUnit` property (application).

## Type of change


## How Can This Be Tested/Reviewed?

Create a new listing with different unit types (e.g studio, 4 bedrooms), then post an application using public and partners. Check if both applications showing properly in partners in the `Household Details > Preferred Unit Sizes `section.


## Checklist:
